definitions/private-law/json/CaseField/ConfidentialityCheck/CaseField.json

```diff
@@ -0,0 +1,122 @@
+[
```

Comment:
Please remove not needed fields, which were created for event based confidential check

Reply:
Done.